cogs/events.py

```diff
@@ -32,11 +33,66 @@ async def on_error(self, event, *args, **kwargs):
             title=event
         )
 
-        if not self.dev_mode:
-            await self.error_hook.send(embed=em)
+        if not constants.DEV_MODE:
+            # await self.error_hook.send(embed=em)
+            await self.client.send_message(embed=em)
         else:
             traceback.print_exc()
 
+    @commands.Cog.listener()
+    async def on_guild_join(self, guild: discord.Guild):
+
+        hw = self.client.user
+        logchannel = self.client.get_channel(constants.log_channel)
+
+        embed = discord.Embed(
+            title="Greetings",
+            description=textwrap.dedent(f"""
+                Thanks for adding HotWired in this server,
+                **HotWired** is a multi purpose discord bot that has Moderation commands, Fun commands, 
+                Music commands and many more!. 
+                The bot is still in dev so you can expect more commands and features.To get a list of commands , 
+                please use **{PREFIX}help** 
+            """),
```

Comment:
I'm not sure `textwrap.dedent` is the right way to do it here, because this function doesn't just remove the new lines, wherever you used new line there will actually be a new line, it will not get merged into a single line.

Reply:
Then, we gotta Organize it!

Reply:
> Then, we gotta Organize it!

I'm not sure what that means, you simply can't use dedent here